crates/pathfinder/src/sequencer/reply.rs

```diff
@@ -240,19 +257,18 @@ pub mod transaction {
     }
 
     /// Represents deserialized L2 transaction receipt data.
+    ///
+    /// As of cairo version 0.7.0 `#[serde(deny_unknown_fields)]`
+    /// is dropped as blocks from around number 49k onwards don't
+    /// contain the `status` field, while the older ones still do.
```

Comment:
Is this true? Surely none of them send the status field now? (I'm expecting this was a code update and that they aren't just storing the raw json blob and sending).

Reply:
Lemme check on some early blocks ;)

Reply:
I checked a few pre-0.7.0 blocks and you're right, I'll remove the comment. Btw I used to get some abnormal replies once in a while in the previous releases when fuzzing so I may have got a bit conditioned to being overly cautious.